packages/calcite-components/src/components/segmented-control/segmented-control.tsx

```diff
@@ -379,8 +378,8 @@ export class SegmentedControl
       this.calciteSegmentedControlChange.emit();
     }
 
-    if (isBrowser() && match) {
-      match.focus();
+    if (!isServer && match) {
+      match?.focus();
```

Comment:
Isn't `match` guaranteed here?

Reply:
Nice catch, that was a leftover from when I initially removed the conditional before trying out `isServer` instead of `isBrowser()`.